### DISCUSSION TITLE
Breadcrumb: Get back from organisation's project to the organisation page

### DISCUSSION THREAD

**Discussion by SquaredP** [community/community#14232] [Projects and Issues]
After accessing a new beta project, created as part of an organisation, I do miss the ability to get back to the organisations page or the projects overview page:

<img width="712" alt="Screenshot 2022-04-04 at 10 23 40" src="https://user-images.githubusercontent.com/47751673/161504144-0d8f17bd-70a7-4f82-8534-43c3e38e8903.png">

Preferably, there would be a breadcrumb like:

`<your org> > <your first project>
`

**Comment by moooifdiw** [comment]
Fffr

**Comment by LydiaWilliams** [comment]
Agreed, moreover I would love there would be: 

`<your org> <your repo> <your project>`

**Reply by SquaredP** [reply]
This might go in as additional requirement, for projects, which are inside a repo.
In my case, the project is inside the org.

You are right: the breadcrumb nav should reflect the path down to the project.
